### DISCUSSION TITLE
Welcome to CodeAnalysis Discussions!

### DISCUSSION THREAD

**Discussion by zhang9w0v5** [Tencent/CodeAnalysis#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mridul1529** [comment]
For pubg
Can we have minimum ping in slowest network?? And can we have a ping detection system that can stop the rendering of a single player.

**Reply by lu599040708** [reply]
ping 服务器吗？  电脑--运行--CMD--ping  服务器IP

**Reply by zhang9w0v5** [reply]
Sorry, we didn't understand this.
Please be more detailed.
thank you!

**Comment by Master-Kang** [comment]
有没有交流群呀   问个问题也找不到人的

**Reply by cyw3** [reply]
不好意思
大家并非全职跟进开源，时间精力比较紧张，我们这个项目是认真执行开源的，不会放着问题不管。

可以先将问题提到issue中，这边会时不时回复issue中的问题哈

**Reply by zhang9w0v5** [reply]
我们请教过一些开源同学，大多数交流群比较不聚焦具体问题。信息量多的容易淹没，也会有不少无效的信息，信息对齐的成本更高，甚至变成水群。
在github上开放式的讨论具体问题利于项目健康发展，我们希望大家遇到过的问题，在github具体问题具体跟进和沉淀，便于后加入的同学学习。

**Comment by Master-Kang** [comment]
cd_error 代码库及帐号不匹配    svn这个报错是什么原因呀，，

**Reply by Lingghh** [reply]
可以提一个issue，附上scmproxy的日志看一下哈
采用docker-compose部署，可以通过``docker-compose logs -f scmproxy``输出节选
采用裸机部署的，可以通过``server/projects/scmproxy/logs/scmproxy.log``文件查阅

**Reply by zhang9w0v5** [reply]
因为scm的复杂度比较高，涉及网络，以及不同版本的git或svn，还有不同的操作系统。
所以，涉及scm的，直接拿log提issue。

**Comment by zhang9w0v5** [comment]
希望大家给予项目更多耐心，共同建设，谢谢！
热烈欢迎成为贡献者！

代码分析系统是比较复杂的综合工程，涉及分布式计算和存储，多操作系统、编译、网络、众多语言垂直细分工具、高IO高CPU高计算负载、信息安全等等......  广度和深度均有。

我们刚开源开放，还需要一段路程将更多梳理好并开放。我们会持续认真的建设该项目。
